### PR TITLE
Bump version to v5.5.5

### PR DESCRIPTION
**Pull request by andreas-bok-sociomantic**
New ABI version: 10.1.5

### CODE REVIEW

**Comment by leandro-lucarella-sociomantic** [comment]
> New ABI version: 10.1.5

Isn't it 11.1.5? That's what it looks like from the diff.

**Comment by andreas-bok-sociomantic** [comment]
> Isn't it 11.1.5? That's what it looks like from the diff.

No, it's black magic version generated by libtool it refers to

File: .travis.yml [hunk 1]
```diff
@@ -69,14 +71,21 @@ script:
...
     provider: script
-    script: beaver bintray upload -N pkg/deb/*.deb
+    script: echo "TRAVIS_TAG='$TRAVIS_TAG'"; beaver bintray upload -N pkg/deb/*.deb
     skip_cleanup: true
     on:
         tags: true
-        condition: >
-            "$TRAVIS_OS_NAME" = "linux" -a
-            "$CC" = "gcc"
+        condition: '"$TRAVIS_OS_NAME" = "linux" && "$CC" = "gcc"'
```
Comment (leandro-lucarella-sociomantic) [inline .travis.yml]: This should be OK, why change it?

Comment (andreas-bok-sociomantic) [reply .travis.yml]: AFAICS because travis wraps the condition in within double brackets and thus expects `&&` as comparison operator.

**Comment by andreas-bok-sociomantic** [comment]
Moved travis script fixes to #188

**Comment by daniel-zullo** [comment]
> Isn't it 11.1.5? That's what it looks like from the diff.
No, it's black magic version generated by libtool it refers to

Maybe you can clarify it in the commit message as I also found it a bit confusing having `LIBDRIZZLE_LIBRARY_VERSION=11:5:1` in the changes and `New ABI version: 10.1.5` in the commit message

**Review by daniel-zullo — COMMENTED** [review]
The changes LGTM. Why is this issue still blocked?

**Comment by andreas-bok-sociomantic** [comment]
There is a pretty detailed description of how the version is bumped in configure.ac, so i would rather just refer to that explanation

**Comment by andreas-bok-sociomantic** [comment]
Should be unblocked, was blocked on #188

**Comment by andreas-bok-sociomantic** [comment]
> There is a pretty detailed description of how the version is bumped in configure.ac, so i would rather just refer to that explanation

Fine by you @daniel-zullo-sociomantic ?

**Comment by andreas-bok-sociomantic** [comment]
Feel free to merge, @daniel-zullo-sociomantic

**Comment by leandro-lucarella-sociomantic** [comment]
I will do the honours then (if there is already a comment describing when to update versions I think it's all fine).

**Comment by leandro-lucarella-sociomantic** [comment]
@andreas-bok-sociomantic let me know when you create the tag so I can check how the upload went.

**Comment by andreas-bok-sociomantic** [comment]
I will ping you

**Pull request merged**